hummingbird/ml/operator_converters/onnx/tree_ensemble.py

```diff
@@ -40,6 +40,9 @@ def _get_tree_infos_from_onnx_ml_operator(model):
             threshold = attr.floats
         elif attr.name == "class_weights" or attr.name == "target_weights":
             values = attr.floats
+            if len(values) == 0:
+                raise TypeError(
+                    "Type mismatch with attribute %r." % attr)
```

Comment:
Nit: in HB we use `string.format()` instead of `%`. Cold you please use format for consistency?

Reply:
done